src/layout.js

```diff
@@ -212,7 +213,7 @@ export function getLengthUnits(length) {
  */
 export function getLengthNumeral(length) {
   const res = parseFloat(length);
-  return isNaN(res) ? undefined : res;
+  return !isFiniteNumber(res) ? undefined : res;
```

Comment:
Technically both `Number` constructor and `parseFloat` can return `Infinity` (both even parse string `'Infinity'` as `Infinity`), although unlikely that the input is `'Infinity'`, I think `isFiniteNumber` is more correct here since the output is expected to be finite. The type check part of  `isFiniteNumber` of course if useless here.